src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_util.py

```diff
@@ -96,7 +104,8 @@ def generic_update_command(self, name, getter_op, setter_op, custom_func_name=No
             self._service_adapter(getter_op),
             self._service_adapter(setter_op),
             factory=self._client_factory,
-            custom_function_op=custom_function_op)
+            custom_function_op=custom_function_op,
+            **kwargs)
```

Comment:
Following is my personal preference, so I don't intend to block this PR here. I usually advise avoiding `**kwargs` if the arguments can be explicit defined. The keyworded arguments are powerful but it also hurts the readability and maintainability. It is not obvious through the glance at the signature to make out what's the additional arguments here. Can we instead of using keyworded arguments define the arguments explicitly?